### PR TITLE
Add custom fonts

### PR DESCRIPTION
**Pull request by jordanwade**
To Test:

Click through app and post any font related suggestions and issues.
Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-22 at 15 32 44

### CODE REVIEW

File: GraceChurch/src/theme/theme.js [hunk 1]
```diff
@@ -113,9 +118,34 @@ const overlays = {
 //   ...propOverrides,
 // };
 const overrides = {
+  H1: {
+    fontFamily: typography.sans.light.default,
+  },
+  H2: {
+    fontFamily: typography.sans.light.default,
+    fontSize: typography.baseFontSize * 1.5,
```
Comment (burkeshartsis) [inline GraceChurch/src/theme/theme.js]: Not sure if this is exactly right but you would probably want to use `rem` here
```suggestion
    fontSize: helpers.rem(1.5),
```

**Review by burkeshartsis — COMMENTED** [review]
if your `fontStack` doesn't use `Inter` or any serif fonts you can safely remove those font files so that we aren't bundling them.

File: GraceChurch/src/theme/theme.js (hunk 1 above)
Comment (jordanwade) [reply GraceChurch/src/theme/theme.js]: 👍 fixed

**Comment by jordanwade** [comment]
Need to clean up Onboarding.

**Review by conrad-vanl — APPROVED** [review]

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**